d3a_interface/sim_results/results_abc.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+from typing import Dict, List
+
+
+class ResultsBaseClass(ABC):
+
+    @staticmethod
+    def _validate_update_parameters(area_result_dict, core_stats, current_market_slot):
```

Comment:
```suggestion
    def _has_update_parameters(area_result_dict, core_stats, current_market_slot):
```
Since it expects to return boolean `True or False` rather than itself raising any exception.

Reply:
Why not, I'll adapt it, thanks!